### DISCUSSION TITLE
Does this work on android or iOS?

### DISCUSSION THREAD

**Discussion by nteko** [heroiclabs/nakama-dart#45] [Q&A] [answered]
I checked flutter_nakama example on web, and it works fine.

but if I try it on the Android and iOS simulator, but it doesn't work at all.

It seems that development is still in progress for Android and iOS.

Or is there anything I need to do before using it on Android or iOS?

Thank you very much.

**Comment by obrunsmann** [comment]
Hi @nteko,

all platforms (Web, Android, iOS, Linux, MacOS and Windows) are supported.

Please send your full stack trace and output of `flutter doctor -v`.

Regards

**Comment by nteko** [comment, marked as answer]
I just found out that when I turn off the ufw, it works on Android.

It does not work if you only open 7351 and 7350 ports, but it works if you disable ufw (open all ports).

I found out 7349 and 7348 ports need to be opened.

AND NOW IT WORKS!

Thank you again.
